pyproject.toml

```diff
@@ -480,6 +480,8 @@ required-imports = ["from __future__ import annotations"]
 "docs/*.py" = ["INP001"]
 "*/decompiled.py" = ["ALL"]
 "ci/release/verify_release.py" = ["T201"] # CLI tool that prints stuff
+"pandas_impl.py" = ["T201"] # prints output using `print`
+"dask_impl.py" = ["T201"] # prints output using `print`
```

Comment:
Should we set this for `docs/**/*.py`?

Reply:
Maybe, though it's certainly possible that spurious prints are in there and they'd be rendered on execution in a `qmd`.

Let's leave it for now and if we starting needing to add this to a bunch of files then we can't blanket ignore `T201` then.